ui/tests/integration/modifiers/code-mirror-test.ts

```diff
@@ -21,6 +21,18 @@ module('Integration | Modifier | code-mirror', function (hooks) {
     assert.dom('.CodeMirror').containsText(valueToChange);
   });
 
+  test('it renders even with undefined value/onInput', async function (assert) {
+    this.set('value', undefined);
+    this.set('onInput', undefined);
+
+    await render(
+      hbs`<div
+      {{code-mirror value=this.value onInput=this.onInput}}
+    />`
+    );
```

Comment:
[sand] may I suggest a tiny pedantic tweak to the indentation 😅 ?

```suggestion
    await render(hbs`
      <div
        {{code-mirror value=this.value onInput=this.onInput}}
      />
    `);
```